src/moin/converters/text_csv_in.py

```diff
@@ -28,7 +28,7 @@ def _factory(cls, type_input, type_output, **kw):
     def __call__(self, data, contenttype=None, arguments=None):
         text = decode_data(data, contenttype)
         content = normalize_split_text(text)
-        dialect = csv.Sniffer().sniff(content[0])
+        dialect = csv.Sniffer().sniff(text)
```

Comment:
so, the sniffer needs more than the first line AND is happy no matter what the line ends are?

Reply:
in my testing, sniffer objects to leading blank lines and input passed as a list.

I have not tested on Ubuntu

Reply:
ok, let's try it.